Add "typings" to package.json

### Summary

I'm using `loadStripe` with Angular, and Angular's build system expects packages to conform to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview#heading=h.k0mh3o8u5hx).  Part of that requirement is to specify the path to the typings file using the "typings" property in the module's package.json.  @stripe/stripe-js has "types" but not "typings", and as a result AOT builds fail.

### Other information
There's a StackBlitz here that demonstrates the error: https://stackblitz.com/edit/angular-kezmsy?file=src%2Fapp%2Fapp.module.ts  The error can be reproduced as follows:

- Download the following project from StackBlitz: https://stackblitz.com/edit/angular-kezmsy?file=src%2Fapp%2Fapp.module.ts
- Extract the zip file: unzip angular-kezmsy.zip
- Install dependencies: npm i
- Do a production build: npm run build -- --prod

That results in a difficult-to-diagnose error:

```
ERROR in ./src/app/app.module.ngfactory.js
Module not found: Error: Can't resolve '@stripe/stripe-js/types/index' in '/home/user/Downloads/angular-kezmsy/src/app'
```
I also files a bug report against angular-cli, which requests that the build process support both "types" and "typings."  The TS doc suggests that the two are synonymous.  Here's that report: https://github.com/angular/angular-cli/issues/17353

And I have a SO question with some additional context: https://stackoverflow.com/questions/60940657/angular-8-and-stripe-js-v3s-loadstripe-module-not-found-error-cant-resolve

Thanks for any help.

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "module": "dist/stripe.esm.js",
   "jsnext:main": "dist/stripe.esm.js",
   "types": "types/index.d.ts",
+  "typings": "types/index.d.ts",
   "scripts": {
     "test": "jest",
     "test:versions": "./tests/versions/scripts/test.sh",

